src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -87,7 +87,7 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
             className="btn btn-primary"
             onClick={() => history.goBack()}
           >
-            Go Back
+            Back
```

Comment:
as I can see this also pops up when user enters a venue, but it's a little different than the other places
guess it should be changed as well tho